otlp/traces.go

```diff
@@ -63,7 +63,7 @@ func TranslateTraceRequest(request *collectorTrace.ExportTraceServiceRequest, ri
 					"span.kind":        spanKind,
 					"name":             span.Name,
 					"duration_ms":      float64(span.EndTimeUnixNano-span.StartTimeUnixNano) / float64(time.Millisecond),
-					"status_code":      statusCode,
+					"span.status_code": statusCode,
```

Comment:
This is a breaking change right? Instead of removing the old value could we add this as an additional value for a period of time?

Reply:
Yeah, it is. I think it may be worth doing the break though, rather than adding the additional field, which may still make things confusing